assets/JWTCSA/0-verification.md

```diff
@@ -0,0 +1,53 @@
+---
+search:
+  exclude: true
+---
+
+# Verification
+
+## `None` Examples
+
+### Java
+
+<!-- --8<-- [start:java] -->
+```
+// HMAC key - Block serialization and storage as String in JVM memory
+private transient byte[] keyHMAC = ...;
+
+...
+
+//Create a verification context for the token requesting
+//explicitly the use of the HMAC-256 hashing algorithm
+JWTVerifier verifier = JWT.require(Algorithm.HMAC256(keyHMAC)).build();
+
+//Verify the token, if the verification fail then a exception is thrown
+DecodedJWT decodedToken = verifier.verify(token);
+```
+<!-- --8<-- [end:java] -->
+
+### Python:pyjwt
+
+<!-- --8<-- [start:pyjwt] -->
+```
+try:
+    pyjwt.decode(encoded, key, algorithms=["HS256","ES256"])
```

Comment:
This is weird to support HS256 (a MAC) and ES256 (a signature) at the same time. Either your key is a shared secret and you wil support HS256 or this is a public key and you support ES256. I think ES256 is spurious here.